transformers4rec/torch/model/prediction_task.py

```diff
@@ -342,7 +344,11 @@ def forward(self, inputs: torch.Tensor, targets=None, training=False, testing=Fa
             # Compute predictions probs
             x, _ = self.pre(x)  # type: ignore
 
-            return x
+            if top_k == -1:
```

Comment:
Does top_k need to be a numeric value? Would it work with with `None` as the default value, for example?

Reply:
I think `-1` is a used convention but we can set it to None either. @sararb will setting it to None create any issue?

Reply:
You're right, it is just a convention. Setting it to None won't create any issue.